### DISCUSSION TITLE
No module named 'nextcord.ext'; 'nextcord' is not a package

### DISCUSSION THREAD

**Discussion by ghost** [nextcord/nextcord#583] [Q&A] [answered]
I've installed nextcord but... Python doesn't seem to recognise it and displays this message:

> "No module named 'nextcord.ext'; 
> 'nextcord' is not a package"

IDE: Python IDLE 3.10.4

![image](https://user-images.githubusercontent.com/103314871/162576733-0a5de82e-ad44-49b4-8b1e-7844badf5f59.png)

I'm clueless on what to do and it seems I've tried everything... any help?

**Comment by Soheab** [comment, marked as answer]
Please rename your folder called "nextcord" to something else and next time please don't send such a small image that no one can read

**Reply by ghost** [reply]
What exactly? Can I just rename it to anything? And apologies for the small image, on my display it was readable.

**Reply by Soheab** [reply]
Yeah it's a folder on your pc, as long as you can find it and it's not "nextcord" it's fine

Oh, on GitHub web it was impossible to read 🤷🏼

**Reply by ghost** [reply]
Huh, I am on GitHub Web and it's decently visible for me.

And also that wasn't the fix, the issue was that the module wasn't in the same folder as the .py file. I'm new to Python so I apologise for any inconvenience caused.

Though, thank you for the reply!
